x-pack/legacy/plugins/siem/public/store/network/model.ts

```diff
@@ -38,36 +38,47 @@ export type TopCountriesTableType =
   | NetworkTableType.topCountriesDestination
   | NetworkTableType.topCountriesSource;
 
+export type TopTlsTableType = IpDetailsTableType.tls | NetworkTableType.tls;
+
 export enum IpDetailsTableType {
   tls = 'tls',
-  topCountriesDestination = 'topCountriesDestinationIp',
-  topCountriesSource = 'topCountriesSourceIp',
-  topNFlowDestination = 'topNFlowDestinationIp',
-  topNFlowSource = 'topNFlowSourceIp',
+  topCountriesDestination = 'topCountriesDestination',
+  topCountriesSource = 'topCountriesSource',
+  topNFlowDestination = 'topNFlowDestination',
+  topNFlowSource = 'topNFlowSource',
   users = 'users',
 }
 
+export type AllNetworkTables = NetworkTableType | IpDetailsTableType;
+
 export interface BasicQueryPaginated {
   activePage: number;
   limit: number;
 }
 
 // Network Page Models
 export interface TopNFlowQuery extends BasicQueryPaginated {
-  topNFlowSort: NetworkTopTablesSortField;
+  sort: NetworkTopTablesSortField;
 }
 
 export interface TopCountriesQuery extends BasicQueryPaginated {
-  topCountriesSort: NetworkTopTablesSortField;
+  sort: NetworkTopTablesSortField;
 }
 
 export interface DnsQuery extends BasicQueryPaginated {
-  dnsSortField: NetworkDnsSortField;
+  sort: NetworkDnsSortField;
   isPtrIncluded: boolean;
 }
 
 export interface TlsQuery extends BasicQueryPaginated {
-  tlsSortField: TlsSortField;
+  sort: TlsSortField;
+}
+
+export interface TableUpdates {
+  activePage?: number;
+  limit?: number;
+  isPtrIncluded?: boolean;
+  sort?: NetworkDnsSortField | NetworkTopTablesSortField | TlsSortField | UsersSortField;
```

Comment:
Consolidating on similar updates like this really ended up refining and cleaning up how we push updates from all the tables. Really nice!